sdk/src/main/scala/com/horizen/api/http/WalletBaseApiRoute.scala

```diff
@@ -50,16 +52,19 @@ abstract class WalletBaseApiRoute[
     withAuth {
       entity(as[ReqCreateKey]) { _ =>
         withNodeView { sidechainNodeView =>
-          //replace to VRFKeyGenerator.generateNextSecret(wallet)
-          val secret = VrfKeyGenerator.getInstance().generateNextSecret(sidechainNodeView.getNodeWallet)
-          val public = secret.publicImage()
-
-          val future = sidechainNodeViewHolderRef ? ReceivableMessages.LocallyGeneratedSecret(secret)
-          Await.result(future, timeout.duration).asInstanceOf[Try[Unit]] match {
-            case Success(_) =>
-              ApiResponseUtil.toResponse(RespCreateVrfSecret(public))
+          val secretFuture = sidechainNodeViewHolderRef ? ReceivableMessages.GenerateSecret(VrfKeyGenerator.getInstance)
```

Comment:
We need to test this API, both with UTs (currently they are not working) and python scripts

Reply:
Unit tests are successful, python tests to be done